src/ngx_postgres_rewrite.c

```diff
@@ -187,7 +187,7 @@
         url_variable.len = 0;
         //fprintf(stdout, "something here %s\n", p);
 
-        while(url_variable.len < (redirect + size) - (p + 1)) {
+        while((int)url_variable.len < (redirect + size) - (p + 1)) {
```

Comment:
@iyesin yes it makes sense to cast the right part. I will, fix that in the pull request, together with title.

Should I close this one and submit a new pull request (I am bit of a newbie with this :) )?